### PR TITLE
Update archer.js

### PR DESCRIPTION
**Pull request by TheWert777**

### CODE REVIEW

File: loc/classes/archer.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ export default {
     es: 'Arquero',
     fr: 'Archer',
     pt: 'Arqueiro'
```
Comment (KaiserCheezel) [inline loc/classes/archer.js]:
```suggestion
    pt: 'Arqueiro',
```

File: loc/classes/archer.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ export default {
...
+    de: 'Bogenschütze',
```
Comment (KaiserCheezel) [inline loc/classes/archer.js]:
```suggestion
    de: 'Bogenschütze'
```

File: loc/classes/archer.js [hunk 3]
```diff
@@ -20,5 +21,6 @@ export default {
     es: 'Los arqueros hacen daño increíble con sus flechas y pueden disparar muchos flechos a una vez con daño explosivo. La estregia mejor es evitar los enemigos y atacar a lo lejos',
     fr: 'Les archers infligent des dégâts élevés sur une cible précise et des dégâts de zone importants. Un bon archer garde toujours ses distances et élimine ses ennemis de loin.',
     pt: 'Arqueiros causam danos altos a um único alvo e danos na Área de Efeito. Um bom arqueiro sempre manterá sua distância e snipe inimigos de longe.'
```
Comment (KaiserCheezel) [inline loc/classes/archer.js]:
```suggestion
    pt: 'Arqueiros causam danos altos a um único alvo e danos na Área de Efeito. Um bom arqueiro sempre manterá sua distância e snipe inimigos de longe.',
```

File: loc/classes/archer.js [hunk 4]
```diff
@@ -20,5 +21,6 @@ export default {
...
+    de: 'Bogenschützen verursachen hohen einzelzielschaden und Berstenden Wirkungsbereich. Ein guter Bogenschütze wird immer Abstand halten und Feinde aus der Ferne schnippen.',
```
Comment (KaiserCheezel) [inline loc/classes/archer.js]:
```suggestion
    de: 'Bogenschützen verursachen hohen einzelzielschaden und Berstenden Wirkungsbereich. Ein guter Bogenschütze wird immer Abstand halten und Feinde aus der Ferne schnippen.'
```

File: loc/classes/archer.js (hunk 2 above)
Comment (TheWert777) [reply loc/classes/archer.js]: I thought you had to have the comma there. I saw everyone else had it by theirs.

Comment (KaiserCheezel) [reply loc/classes/archer.js]: No, you don't put commas at the end of the last translation, however you do put them at the end of the other translations.

**Pull request merged**